src/mouse/default_gutter_handler.js

```diff
@@ -151,18 +151,27 @@ class GutterTooltip extends Tooltip {
         Tooltip.prototype.setPosition.call(this, x, y);
     }
     
-    static get annotationLabels() { return {
+    static get annotationLabels() {
+        return {
             error: {
-                singular: nls("gutter-tooltip.aria-label.error.singular", "error"), 
+                singular: nls("gutter-tooltip.aria-label.error.singular", "error"),
                 plural: nls("gutter-tooltip.aria-label.error.plural", "errors")
             },
+            security: {
```

Comment:
We should also add these new strings to https://github.com/ajaxorg/ace/blob/master/src/lib/default_english_messages.js#L44

I think that should work automatically if you run `node Makefile.dryice.js nls`